packages/contracts/src/contracts/current/protocol/Exchange/libs/LibOrder.sol

```diff
@@ -98,6 +98,26 @@ contract LibOrder is
         bytes32 schemaHash = EIP712_ORDER_SCHEMA_HASH;
         bytes32 makerAssetDataHash = keccak256(order.makerAssetData);
         bytes32 takerAssetDataHash = keccak256(order.takerAssetData);
+        // Assembly for more efficiently computing:
+        // keccak256(abi.encodePacked(
+        //   DOMAIN_SEPARATOR_SCHEMA_HASH,
+        //   keccak256(abi.encodePacked(address(this))),
+        //   ORDER_SCHEMA_HASH,
+        //   keccak256(abi.encodePacked(
```

Comment:
`abi.encode` not `abi.encodePacked` (which would produce a different hash value).